WNPRC_EHR/src/org/labkey/wnprc_ehr/notification/WaterMonitoringNotification.java

```diff
@@ -66,7 +66,7 @@ public WaterMonitoringNotification(Module owner)
     @Override
     public String getName()
     {
-        return "Water Monitoring";
+        return "Water Monitoring 2";
```

Comment:
remove this addition

Reply:
Not sure why I changed this, my bad.  I think I was doing some testing with it before I began the Death Notification.